tests/succeed/ann/fun-literal-identity-poly-sugar.snap

```diff
@@ -1,4 +1,4 @@
 stdout = '''
-fun A a => a : fun (A : Type) -> A -> A
+fun (A : Type) (a : a) => a : fun (A : Type) -> A -> A
```

Comment:
Looks like a scoping error has been introduced somewhere?
Should be `fun (A : Type) (a : A) => a : fun (A : Type) -> A -> A`

Reply:
ooooh yes you are right. wouldn't it be great if there was a way to _ensure_ these were well scoped…

Reply:
I think it might be to do with when the function parameter types are quoted but I'll have a look

Reply:
Was actually an issue in the distiller, which would have been a bit trickier to catch, alas. Perhaps with some sort of roundtrip test maybe.